metadata-ingestion/src/datahub/ingestion/source/confluent_schema_registry.py

```diff
@@ -5,6 +5,7 @@
 from typing import Any, List, Optional, Set, Tuple
 
 import confluent_kafka
+import confluent_kafka.schema_registry.schema_registry_client
```

Comment:
Would prefer to do a `from ... import` like we do for `RegisteredSchema`, `Schema`, `SchemaReference`

Reply:
updated